packages/@ama-sdk/schematics/README.md

```diff
@@ -64,27 +64,49 @@ npx -p @angular/cli ng add @ama-sdk/core
 
 ### How to use?
 
-The typescript generator provides 2 generators:
+The typescript generator provides 3 generators:
 
 - **shell**: To generate the "shell" of an SDK package
-- **core**: To (re)generate the SDK based on a specified Swagger spec
+- **core**: To (re)generate the SDK based on a specified OpenApi specification
 - **create**: To create a new SDK from scratch (i.e. chain **shell** and **core**)
 
-To generate the `shell` you can run:
+To generate the `shell` in an existing monorepo, you can run:
 
 ```shell
 yarn schematics @ama-sdk/schematics:typescript-shell
 ```
 
-If you use `Yarn2+`, you can use the following `scripts` in `package.json`:
+or from scratch, with the NPM initializer:
 
-```json
-    "resolve": "node -e 'process.stdout.write(require.resolve(process.argv[1]));'",
-    "generate": "yarn schematics @ama-sdk/schematics:typescript-core --spec-path ./swagger-spec.yaml",
-    "upgrade:repository": "yarn schematics @ama-sdk/schematics:typescript-shell",
+```shell
+npm create @ama-sdk typescript <project-name>
 ```
 
-Use `generate` to (re)generate your SDK based on the content of `./swagger-spec.yaml` (make sure you have this file at the root of your project) and `upgrade:repository` to regenerate the structure of your project.
+The package generated comes with the following script in the package.json:
+
+```json5
+{
+  // ...
+  "generate": "yarn schematics @ama-sdk/schematics:typescript-core --spec-path ./openapi.yaml",
+  "upgrade:repository": "yarn schematics @ama-sdk/schematics:typescript-shell"
+}
+```
+
+> [!NOTE]
+> Use `generate` to (re)generate your SDK based on the content of `./swagger-spec.yaml` (make sure you have this file at the root of your project) and `upgrade:repository` to regenerate the structure of your project.
```

Comment:
openapi.yaml instead of swagger-spec.yaml no? should be the same file name as in the generate command line above?